api/src/services/validation-service.ts

```diff
@@ -101,7 +104,7 @@ export class ValidationService extends DBService {
       await this.parseDWCToJSON(submissionId, dwcPrep.archive);
 
       // TODO: scraping refactor https://quartech.atlassian.net/browse/BHBC-2098
```

Comment:
Any chance we can delete this, seeing as how this ticket should address the intent behind the comment?